tests/test_opSimOrdering.py

```diff
@@ -0,0 +1,76 @@
+import os
```

Comment:
Yes, I think just using the small OpSim file is the way to go here.  Even doing `setUpClass` really won't be enough of a speed up.